### PR TITLE
Shorten reserve and capture. Refer to common topics

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
@cloveras - This one was a bit trickier.

**Comment by cloveras** [comment]
Looks good. Some details are eCom specific, so @ananthb86 can have a look after the merge to see if we need to add a little eCom/Recurring specific details on the "common topics" page.

**Review by cloveras — APPROVED** [review]

**Comment by cloveras** [comment]
Slack: https://vippsas.slack.com/archives/C03FX291T7H/p1667394331933139

File: vipps-ecom-api.md [hunk 1]
```diff
@@ -925,75 +922,22 @@ See FAQ:
...
-We strongly recommend that you use the idempotency key, `X-Request-Id`, in the capture call. Then, if a capture
+We strongly recommend that you use the idempotency key,
+[X-Request-Id](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/capturePaymentUsingPOST)
+in the capture call. Then, if a capture
 request fails for any reason, it can be retried with the same idempotency key.
-You can use any unique id for your `X-Request-Id`.
-
-See the [X-Request-Id in the capturePaymentUsingPOST specification](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/capturePaymentUsingPOST) for details.
-
-To perform a normal capture of the entire amount, `amount` can be
```
Comment (cloveras) [inline vipps-ecom-api.md:945]: This is quite specific for the eCom API.

File: vipps-ecom-api.md [hunk 2]
```diff
@@ -925,75 +922,22 @@ See FAQ:
...
+For more details, see
+[Common topics: Reserve and capture](https://github.com/vippsas/vipps-developers/blob/master/common-topics/reserve-and-capture.md).
 
 ## Capture
 
 Capture is done with
 [`POST:/ecomm/v2/payments/{orderId}/capture`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/capturePaymentUsingPOST).
 
-We strongly recommend that you use the idempotency key, `X-Request-Id`, in the capture call. Then, if a capture
+We strongly recommend that you use the idempotency key,
```
Comment (tht13) [inline vipps-ecom-api.md:933]: Lets just say you have to send it (yes I know its optional, but better to make people think its required)

**Comment by rebekaburnett** [comment]
Let's talk about this in person

**Comment by rebekaburnett** [comment]
@tht13 - Can you help me with this. I have added most everything back into the ecom  API section about reserve and capture. 

Meanwhile, I tried to make the "common topics" version more general - see https://github.com/vippsas/vipps-developers/blob/master/common-topics/reserve-and-capture.md

Can you help me find the right balance here? The problem is that they are repeating a lot of the same stuff, but I feel like it is useful for the user to read that stuff directly in the ecom guide, too, so it's hard to remove it. You have a better feeling for it.
Here's a PR where you can delete stuff from common topics, if you want:  https://github.com/vippsas/vipps-developers/pull/226

**Review by tht13 — APPROVED** [review]

**Review by tht13 — APPROVED** [review]

**Pull request merged**